### PR TITLE
Storage scenarios

### PR DESCRIPTION
**Pull request by AliBayatGH**
Creates an identity box (IdBox).
Creates a seed identity as the first primary identity.
Creates additional seed identities.
Updates an existing identity with added key information
Updates an existing identity with identity info.
Reads identity information given an identifier

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,388 @@
-bin/
```
Comment (nimakam) [inline .gitignore:1]: Should we be deleting this? Is it included in the below?
Why add all this below? Can we just add small bits as we go, keep it simple and add complexity progressively?

Comment (nimakam) [inline .gitignore:1]: Should we be deleting this? Is it included in the below?
Why add all this below? Can we just add small bits as we go, keep it simple and add complexity progressively?

**Review by nimakam — COMMENTED** [review]
Everything looks great. Except I'm not sure about the gitignore stuff. Normally I just exclude a few root folder names like bin/ obj/ .vs/ .vscode/ and files like .suo, and that covers everything

File: .gitignore (hunk 1 above)
Comment (AliBayatGH) [reply .gitignore:1]: I usually use this file, which is the default Visual Studio template.
But whatever you see fit. I will do.
https://github.com/github/gitignore/blob/master/VisualStudio.gitignore

Comment (nimakam) [reply .gitignore:1]: If it’s ok let’s just add the things we use and not everything. I’m assuming they have this as a reference to pick and choose from.

Also I use VSCode and others may use other tools, so they can also add bits and it’s all discoverable and manageable.

**Comment by nimakam** [comment]
Excited to see all your new changes as part of the pull request and eventually merged! 👌

**Review by nimakam — APPROVED** [review]

**Pull request merged**